packages/server/src/api/controllers/table/internal.ts

```diff
@@ -44,7 +44,9 @@ function checkAutoColumns(table: Table, oldTable?: Table) {
 export async function save(ctx: UserCtx<SaveTableRequest, SaveTableResponse>) {
   const db = context.getAppDB()
   const { rows, ...rest } = ctx.request.body
-  let tableToSave: TableRequest = {
+  let tableToSave: Table & {
```

Comment:
Not sure why this change. The old type should be compatible

Reply:
Me neither, but I think it was maybe a commit you made yourself?
![image](https://github.com/Budibase/budibase/assets/9075550/16b35f3c-4665-4fac-8b37-b6285b811504)

Reply:
Unless maybe this changed again with your recent views permission code? According to my git history it was never of type `TableRequest`. This was maybe changed upstream in develop?
![image](https://github.com/Budibase/budibase/assets/9075550/39bf2e29-bb00-42d1-9d02-83cf6cff5f48)

Reply:
Funny 😅 I will have a look at it